app/controllers/catalog_controller.rb

```diff
@@ -673,4 +673,26 @@ def index
   rescue ActionController::BadRequest
     render file: Rails.public_path.join('x400.html'), layout: true, status: :bad_request
   end
+
+  def show
+    # Let Blacklight do the normal flow
+    value = super
+
+    host_id = nil
+    if @document.id == "9929455793506421" || @document["contained_in_s"]
+      # This value should come from @document["contained_in_s"] but right now the value
+      # it's empty because we are indexing subfield x instead of subfield w.
+      host_id = "9929455793506421" || @document["contained_in_s"]
```

Comment:
We are indexing 773w. https://github.com/pulibrary/bibdata/blob/35e81e844b6491cdedd65f20313838364eaf31b7/marc_to_solr/lib/traject_config.rb#L586